### DISCUSSION TITLE
powering mini computer

### DISCUSSION THREAD

**Discussion by arielc-brillianetor** [iRobotEducation/create3_docs#232] [Questions]
Hi,
I wanted to know if I connected a raspberry pi or a jetson nano to the create3, does the create3 provide enough power or do I need a battery for the computer as well?
Thanks a lot,
Ariel

**Comment by jacobwaller** [comment]
It provides plenty of power for a raspberry pi

**Comment by shamlian** [comment]
Hi Ariel,

The Create 3 is able to provide a regulated 5 V @ 3 A from a 2 A budget of the battery voltage. For details, see the [page on the adapter board](https://iroboteducation.github.io/create3_docs/hw/adapter/#power-connections). The robot has a 26Wh battery, so you can calculate runtime by adding the power draw of your single board computer of choice with the robot's average power draw (which is dependent on how much moving around you're planning on doing while off-changer), and dividing this by the total energy available.

Hookup diagrams for both [Pi](https://iroboteducation.github.io/create3_docs/hw/rpi_hookup) and [Nano](https://iroboteducation.github.io/create3_docs/hw/jetson_hookup/) are available. You can search these discussions to find other users who use both devices. I personally keep a Raspberry Pi 4 plugged into my Create 3 without any trouble.

--Steve
